homepage/de.js

```diff
@@ -0,0 +1,74 @@
+{
+  "Integrations": "Integrationen",
+  "Pricing": "Preise",
+  "Apps": "Apps",
+  "Existing user login": "Bestehende Benutzer-Anmeldung",
```

Comment:
How about: "Als bestehender Nutzer anmelden" like in [#13](https://github.com/gitterHQ/gitter-translations/pull/13/files#diff-02133d9a97f86dc218a62446d3e99843R5)?